src/Pelagos/Bundle/AppBundle/Resources/views/MetadataGenerator/MD_Distribution.xml.twig

```diff
@@ -108,18 +108,20 @@
                             <gmd:protocol>
                                 <gco:CharacterString>https</gco:CharacterString>
                             </gmd:protocol>
-                            <gmd:name>
-                                <gco:CharacterString>Data Landing Page</gco:CharacterString>
-                            </gmd:name>
-                            <gmd:description>
-                                <gco:CharacterString>GRIIDC dataset landing page</gco:CharacterString>
-                            </gmd:description>
-                            <gmd:function>
-                                <gmd:CI_OnLineFunctionCode codeList="http://www.isotc211.org/2005/resources/Codelist/gmxCodelists.xml#CI_OnLineFunctionCode"
-                                                           codeListValue="information" codeSpace="002">
-                                    information
-                                </gmd:CI_OnLineFunctionCode>
-                            </gmd:function>
+                            {%  if dataset.datasetsubmission.datasetFileTransferStatus != constant('\Pelagos\\Entity\\DatasetSubmission::TRANSFER_STATUS_REMOTELY_HOSTED') %}
```

Comment:
I think we should use availability status for remotely hosted datasets.

Reply:
This is a good catch. Similarly also need to consider both restricted and unrestricted remotely hosted cases.